builtin/k8s/platform.go

```diff
@@ -120,6 +120,18 @@ func (p *Platform) Deploy(
 	step.Update("Kubernetes client connected to %s with namespace %s", config.Host, ns)
 	step.Done()
 
+	step = sg.Add("Validating cluster...")
+
+	nodes, err := clientSet.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+	if err != nil {
+		step.Update("Unable to read node list, assuming cluster healthy")
```

Comment:
I think this can be a weird message. I think we should just ignore the error (maybe log it) since this whole thing is a best effort check anyways.